kclvm/runner/src/lib.rs

```diff
@@ -286,3 +266,29 @@ fn temp_file(dir: &str) -> String {
     std::fs::create_dir_all(dir).unwrap_or_else(|_| panic!("{} not found", dir));
     Path::new(dir).join(file).to_str().unwrap().to_string()
 }
+
+pub fn join_work_path_with_k_file(
```

Comment:
Add more comments. Besides, put it into another crate, such as `driver` or `bootstrap`?